2_FirstComponent.md

```diff
@@ -154,16 +108,30 @@ The `scope` object is the can.Component's view model. The view model is an abstr
 ![](images/2_first_component/ComponentScopeTemplateLink.png)
 
 #####can.Map &amp; can.List
-The scope is a special type of object, called a "can.Map". can.Map objects are observable. Observable objects provide a way for you to listen for and keep track of changes to them. What this means, in this instance, is that if you make a change to your scope, those changes will be reflected automatically in your view. If you've cross-bound the values between your scope and your view, changes to your view will also be reflected in your scope. We'll see how this works in the next chapter.
+The scope is a special type of object, called a "can.Map". can.Map objects are observable. Observable objects emit events whenever their properties change. can.view (and subsequently can.Component) subscribe to these events and update the DOM as needed. If you've cross-bound the values between your scope and your view, changes to your view will also be reflected in your scope. We'll see how this works in the next chapter.
+
+can.Map is able to emit these events because of the `attr` method. This is important. In order to broadcast the associated events when you change a property on a `can.Map`, you must use the `attr` method when setting or getting a value.
+
+The `attr` method can be used to either get or set a property on a can.Map. 
+      
+    var myCanMapInstance = new can.Map({
+      id: 12,
+      person: {
+        name: {
+          last: 'Ludwick'
+          last: 'Beethoven'
+        }
+      } 
+    });
```

Comment:
I felt like going into "dot" notation/deep properties was a bit of a leap from the first paragraph, so I added this example. But now both of these examples seem out of place.